sources/packages/web/src/router/StudentRoutes.ts

```diff
@@ -168,6 +175,56 @@ export const studentRoutes: Array<RouteRecordRaw> = [
           clientType: ClientIdType.Student,
         },
       },
+      {
+        path: AppRoutes.StudentApplicationOfferingChangeRequest,
+        name: StudentRoutesConst.STUDENT_APPLICATION_OFFERING_CHANGE_REQUEST,
```

Comment:
It will be better if you could add `  redirect: ` to the parent

Reply:
@ann-aot do you mean `redirect:` to the child?

Reply:
yes @sh16011993 , I believe that's what we discussed in the call